modules/nf-core/pilon/tests/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process PILON"
+    script "../main.nf"
+    process "PILON"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pilon"
+
+
+    test("Test PILON") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:false ], // meta map
+                            file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
+                            ]
+                input[1] = [ [ id:'test', single_end:false ], // meta map
+                               file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                               file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true),
+                            ]
+                input[2] = "bam"
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
```

Comment:
I see that the module has multiple modes, is it possible to add tests for the others? And a stub test as well, although I see the module currently lacks a stub